package.json

```diff
@@ -1,24 +1,42 @@
 {
-  "name": "@ni/javascript-styleguide",
-  "version": "1.0.0-alpha.0",
-  "description": "NI's JavaScript and TypeScript style guide",
-  "private": true,
+  "name": "@ni/eslint-config",
+  "version": "1.0.0-alpha.2",
+  "description": "NI's ESLint Shareable Config",
+  "main": "index.js",
   "scripts": {
-    "eslint-config:install": "cd packages/eslint-config && npm install",
-    "eslint-config:publish": "cd packages/eslint-config && npm publish",
-    "eslint-config:test": "cd packages/eslint-config; npm test",
-    "preinstall": "npm run eslint-config:install",
-    "publish:packages": "npm run eslint-config:publish",
-    "test": "npm run eslint-config:test"
+    "prepare": "npm test",
+    "test": "eslint ."
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ni/javascript-styleguide.git"
   },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "ni"
+  ],
   "author": "NI",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/ni/javascript-styleguide/issues"
   },
-  "homepage": "https://github.com/ni/javascript-styleguide#readme"
+  "homepage": "https://github.com/ni/javascript-styleguide#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    ".eslintrc.js"
```

Comment:
Add explicit file include list so other files added to repo are ignored (License, README, pacakge.json automatically pulled in by npm)

Reply:
Does npmjs.com use the GitHub README or the package's? The [documentation](https://docs.npmjs.com/about-package-readme-files) just says, "we recommend including a README file in your package directory."

Reply:
Yep! The linked page states "The README file will be shown on the package page."